raw-data/UW/uw_inventories.tsv

```diff
@@ -0,0 +1,55 @@
+				d				
+				ʔ	"restricted to word-medial contexts"			
+				ˀb	"there is a series of pre-glottalized voiced consonants that can appear both word-initially and word-medially."	ˀb	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				ˀb		ˀm	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				ˀd	"there is a series of pre-glottalized voiced consonants that can appear both word-initially and word-medially."	ˀd	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				ˀd		ˀn	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				h				
+				hʷ				
+				ts		ts	"This variation seems to be both intra- and inter-speaker and more research is needed to understand what factors motivate the choice of one of the three allophones of /ʦ/."	
+				ts		s	"This variation seems to be both intra- and inter-speaker and more research is needed to understand what factors motivate the choice of one of the three allophones of /ʦ/."	
+				ts		ʃ	"This variation seems to be both intra- and inter-speaker and more research is needed to understand what factors motivate the choice of one of the three allophones of /ʦ/."	
+				ts		ɲ̥	"The [ɲ̥]... occurs in the realization of nouns formed with the dummy root iʦ- when the classifier attached to the root is nasal."	
+				t̠ʃ				
+				d̠ʒ		d̠ʒ	"The palatal nasal [ɲ] is an allophone of the /ʤ/ when this sound is followed by a nasal."	
+				d̠ʒ		ɲ	"The palatal nasal [ɲ] is an allophone of the /ʤ/ when this sound is followed by a nasal."	
+				d̠ʒ		j	"The consonants /ʤ/ and /b/ can undergo lenition and be realized as [j] and [w] respectively. The context for /ʤ/ lenition seems to be before front vowels"	
+				ˀd̠ʒ	"there is a series of pre-glottalized voiced consonants that can appear both word-initially and word-medially."	ˀd̠ʒ	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				ˀd̠ʒ		ˀɲ	"the pre-glottalized nasals only occur when the following vowel is a nasal and the pre-glottalized oral stops when the following vowel is oral"	
+				m				
+				n				
+				l		l	"The liquid phoneme has two realizations: [ɾ] in front of /i/ and /ĩ/ in word-medial position and [l] in all other contexts."	
+				l		ɾ	"The liquid phoneme has two realizations: [ɾ] in front of /i/ and /ĩ/ in word-medial position and [l] in all other contexts."	
+				l		n	"/l/ can also be variably nasalized when followed by nasal vowels"	
+				w				
+				ˀw				
```

Comment:
GitHub complains when files don't end with blank lines. You may need to open the .tsv in a plain text editor (rather than spreadsheet editor) to make that happen.